Category.h

```diff
@@ -0,0 +1,18 @@
+//
+//  Category.h
+//  OptionSelector
+//
+//  Created by Umar on 8/13/15.
+//  Copyright © 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface CQCategory : NSObject
```

Comment:
It's good practice to include at least one comment for each class describing what it does or what data it stores (if it's just a data storage class). Even if it seems obvious to you now, it might not in a month (and it might not to your colleagues).